ckan/tests/controllers/test_feed.py

```diff
@@ -1,147 +1,95 @@
-                         id=group['name']) + '?page=abc'
+        offset = url_for(u'feeds.group', id=group['name']) + '?page=abc'
         app = self._get_test_app()
+        offset = url_for(u'feeds.group', id=group['name']) + '?page=abc'
         res = app.get(offset, status=400)
         assert '"page" parameter must be a positive integer' in res, res
 
     def test_general_atom_feed_works(self):
         dataset = factories.Dataset()
-        offset = url_for(controller='feed', action='general')
+        offset = url_for(u'feeds.general')
         app = self._get_test_app()
+        offset = url_for(u'feeds.general')
         res = app.get(offset)
 
-        assert '<title>{0}</title>'.format(dataset['title']) in res.body
+        assert u'<title type="text">{0}</title>'.format(
+            dataset['title']) in res.body
 
     def test_group_atom_feed_works(self):
         group = factories.Group()
         dataset = factories.Dataset(groups=[{'id': group['id']}])
-        offset = url_for(controller='feed', action='group',
-                         id=group['name'])
+        offset = url_for(u'feeds.group', id=group['name'])
         app = self._get_test_app()
+        offset = url_for(u'feeds.group', id=group['name'])
         res = app.get(offset)
 
-        assert '<title>{0}</title>'.format(dataset['title']) in res.body
+        assert u'<title type="text">{0}</title>'.format(
+            dataset['title']) in res.body
 
     def test_organization_atom_feed_works(self):
         group = factories.Organization()
         dataset = factories.Dataset(owner_org=group['id'])
-        offset = url_for(controller='feed', action='organization',
-                         id=group['name'])
+        offset = url_for(u'feeds.organization', id=group['name'])
         app = self._get_test_app()
+        offset = url_for(u'feeds.organization', id=group['name'])
         res = app.get(offset)
 
-        assert '<title>{0}</title>'.format(dataset['title']) in res.body
+        assert u'<title type="text">{0}</title>'.format(
+            dataset['title']) in res.body
 
     def test_custom_atom_feed_works(self):
         dataset1 = factories.Dataset(
-            title='Test weekly',
-            extras=[{'key': 'frequency', 'value': 'weekly'}])
+            title=u'Test weekly',
+            extras=[{
+                'key': 'frequency',
+                'value': 'weekly'
+            }])
         dataset2 = factories.Dataset(
-            title='Test daily',
-            extras=[{'key': 'frequency', 'value': 'daily'}])
-        offset = url_for(controller='feed', action='custom')
-        params = {
-            'q': 'frequency:weekly'
-        }
-        app = self._get_test_app()
-        res = app.get(offset, params=params)
-
-        assert '<title>{0}</title>'.format(dataset1['title']) in res.body
-
-        assert '<title>{0}</title>'.format(dataset2['title']) not in res.body
-
-
-class TestFeedInterface(helpers.FunctionalTestBase):
```

Comment:
We seem to have lost this test class? It's important as it tests a plugin interface, which we want to make sure keeps working as until now

Reply:
i remove those tests as i had problems to find/rewrite replacement for [webhelpers GeoAtom1Feed](https://bitbucket.org/bbangert/webhelpers/src/9ad434bec9a16c06c1cfeed38cde02f00a95685d/webhelpers/feedgenerator.py?at=trunk&fileviewer=file-view-default#feedgenerator.py-550).

Reply:
@tino097 just change [this](https://github.com/tino097/ckan/blob/f1f36f8a67b1e247005bb7cb9911b42ff8227af8/ckan/tests/controllers/test_feed.py#L140) to use whatever constructor class you are using in the feeds blueprint now.